cmd/run/run.go

```diff
@@ -448,6 +460,10 @@ func ReadConfig() (*Config, error) {
 }
 
 func VerifyConfig(cfg *Config) error {
+	if int(cfg.MaxConcurrentReads) > cfg.Datastore.MaxOpenConns {
```

Comment:
Why so strict here? If max_concurrent_reads is higher than max_open_conns there isn't anything inherently wrong with that. It should be a `log.Warn` at most, because it _could_ mean that RPCs will get throttled a bit waiting for connections, but it's not an error.

Reply:
My thinking here was that, say they set `maxOpenConns = 1` and `maxConcurrentReads = 100`. `maxOpenConns` will still dominate, no? And no more than 1 request at a time will go through. So my goal is to not give a false idea of what the performance will be like

Reply:
Yeah, but that doesn't need to be an error. Just `log.Warn`. An error implies that the system will not work if you have the values the way they are.